extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +58,232 @@ limitations under the License.
+    - For instance, this would be a valid use of toolbar: `<nav toolbar="(max-width: 1024px)">`
+- The nav containing the toolbar attribute, must only contain a single `<ul>` element, that contains `<li>` elements
+    - The `<li>` elements may may contain any valid HTML elements (supported by AMP), or any of the AMP elements that `<amp-sidebar>` supports.
+- The nav element, or it's `<ul>`'s `<li>` elements, may also contain the attribute `toolbar-only`
+    - The attribute `toolbar-only` will hide the elements with the attribute in the sidebar, but leave them shown in the toolbar
+- The nav element may also contain the attribute `target` that accepts an element id
+    - The attribute `target` will place the toolbar into the specified id of the element on the page, without the default toolbar styling.
+    - e.g `<nav toolbar="(max-width: 1024px)" target="toolbar-target">`
+- Toolbar behavior is only applied while the media-query is valid
 
-**layout**
-
-The only permissible value for the `layout` attribute in `amp-sidebar` is `nodisplay`.
+Example:
 
-**common attributes**
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
-This element includes [common attributes](https://www.ampproject.org/docs/reference/common_attributes) extended to AMP components.
+  <nav toolbar="(max-width: 767px)">
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Styling
+Example (toolbar-only on `<nav>` element):
 
-The `amp-sidebar` component can be styled with standard CSS.
+```html
+<amp-sidebar id="sidebar1" layout="nodisplay" side="right">
+  <ul>
+    <li> Nav item 1</li>
+    <li> Nav item 2</li>
+    <li> Nav item 3</li>
+    <li> Nav item 4</li>
+    <li> Nav item 5</li>
+    <li> Nav item 6</li>
+  </ul>
 
-- The `width` of the `amp-sidebar` may be set to adjust the width of the sidebar between the pre-set min(45px) and max(80vw) values.
-- The height of the `amp-sidebar` may be set to adjust the height of the sidebar if required. If the height exceeds 100vw then the sidebar will have a vertical scrollbar. The preset height of the sidebar is 100vw and can be overridden in CSS to make it shorter.
-- The current state of the sidebar is exposed via the `open` attribute that is set on the `amp-sidebar` tag when the side bar is open on the page.
+  <nav toolbar="(min-width: 0px)" toolbar-only>
+    <ul>
+      <li>
+        <input placeholder="Search..."/>
+      </li>
+    </ul>
+  </nav>
+</amp-sidebar>
+```
 
-## Actions
-The `amp-sidebar` exposes the following actions you can use [AMP on-syntax to trigger](../../../spec/amp-actions-and-events.md):
+Example (toolbar-only on individual `<li>` elements):
```

Comment:
- Rewrite as "Example: Displaying specific items only in the toolbar"

This example confuses me, why specify "toolbar-only" on an element that's already only in the toolbar (it's a new input that doesn't appear in the sidebar.  Isn't this example essentially the same as the 1st basic example?
 Or, do additional items added to the toolbar also appear in the sidebar? This needs clarification here  and up above where behavior is.